lib/pbench/cli/server/tree_manage.py

```diff
@@ -18,34 +111,49 @@ def print_tree(tree: CacheManager):
     print("Tarballs:")
     for tarball in tree.datasets.values():
         print(f"  {tarball.name}")
+        if tarball.unpacked:
+            date = datetime.fromtimestamp(
+                tarball.last_ref.stat().st_mtime, timezone.utc
+            )
+            print(f"    Inventory is cached, last referenced {date:%Y-%m-%d %H:%M:%S}")
 
     print("\nControllers:")
     for controller in tree.controllers.values():
         print(f"  Controller {controller.name}:")
         for tarball in controller.tarballs.values():
             print(f"    Tarball {tarball.name}")
-            if tarball.unpacked:
-                print(f"      Unpacked in {tarball.unpacked}")
 
 
 @click.command(name="pbench-tree-manager")
 @pass_cli_context
 @click.option(
     "--display", default=False, is_flag=True, help="Display the full tree on completion"
 )
+@click.option(
+    "--lifetime",
```

Comment:
I really implemented this so I could do some manual testing with `--lifetime 0.001` to watch the server through multiple cycles without waiting; but I also left it running overnight to be sure that the normal 4 hour timer mode reclaim worked. I don't imagine this option will be used often, but it could be handy to be able to manually prune the cache if, say, we notice that the disk is clogged up with a lot of short term cache activity.

I also had the idea that the default cache lifetime might be influenced by available disk space ... and deliberately avoided thinking too much about that in this first pass.